core/components/commerce_digitalproduct/src/Modules/Digitalproduct.php

```diff
@@ -33,6 +34,7 @@ public function initialize(EventDispatcher $dispatcher)
         $this->adapter->loadLexicon('commerce_digitalproduct:default');
 
         $dispatcher->addListener(\Commerce::EVENT_CHECKOUT_AFTER_STEP, [$this, 'addCheckoutPlaceholders']);
+        $dispatcher->addListener(\Commerce::EVENT_ORDER_PLACEHOLDERS, [$this, 'addGetOrderPlaceholders']);
```

Comment:
Should add a conditional here to make sure that constant exists as it's only added later and otherwise break the module entirely

Reply:
Done